parsec/core/gui/workspaces_widget.py

```diff
@@ -618,8 +676,8 @@ def reencrypt_workspace(
         if r != _("ACTION_WORKSPACE_REENCRYPTION_CONFIRM"):
             return
 
-        @contextmanager
-        def _handle_fs_errors():
+        @contextmanager  # type: ignore[arg-type]
+        def _handle_fs_errors() -> Iterable[None]:
```

Comment:
Shouldn't it be AsyncGenerator ?

Reply:
No, the function isn't async. After digging a bit more it turns out we can return an `Iterator[None]`  because `contextmanager` decorator expects an `Iterator` object